### PR TITLE
Add gradient as background for images

### PR DESCRIPTION
**Pull request by KomalBhadani27**
**Gradient of top 3 dominant colors is added as background**

**When Cloud Vision API fails to return dominant colors**

### CODE REVIEW

**Comment by telescopic** [comment]
Looks amazing! :D great work 👍

File: stamp_generation/stamp_templates/stamp_base_template.jj2 [hunk 1]
```diff
@@ -51,7 +51,7 @@
                 {% if contents[page.media_index].content_type.name == 'EMBEDDED_PINTEREST_PIN'-%}
                     {%- include "embedded_pinterest_pin_template.jj2" -%}
                 {%- endif -%}
-                {% if not page.is_embedded_content %}
+                {% if not (page.is_embedded_content or contents[page.media_index].content_type.name == 'QUOTE') %}
```
Comment (detel) [inline stamp_generation/stamp_templates/stamp_base_template.jj2:54]: Its better to use functions. We should minimize logic in jinja files.

**Comment by detel** [comment]
Do we need background for text now?

**Comment by KomalBhadani27** [comment]
> Do we need background for text now?

yes, because we still don't know where text would appear on image. So if image size is more in height, then we would need it.

File: stamp_generation/stamp_templates/stamp_base_template.jj2 (hunk 1 above)
Comment (KomalBhadani27) [reply stamp_generation/stamp_templates/stamp_base_template.jj2:54]: Makes sense! Thanks for suggestion. Will be updated in next iterations. Merging this PR.

**Pull request merged**